data/src/main/java/org/apache/iceberg/data/orc/GenericOrcReader.java

```diff
@@ -236,11 +332,7 @@ public BigDecimal convert(ColumnVector vector, int row) {
       Preconditions.checkArgument(icebergField.type().isListType());
       TypeDescription child = schema.getChildren().get(0);
 
-      childConverter = buildConverter(icebergField
-          .type()
-          .asListType()
-          .fields()
-          .get(0), child);
+      childConverter = buildConverter(icebergField.type().asListType().fields().get(0), child);
```

Comment:
I prefer this style, but it's better not to make non-functional changes like this to avoid commit conflicts.

Reply:
Reverted